src/main/java/org/fundacionjala/at15/pokemon/Battle.java

```diff
@@ -1,5 +1,8 @@
 package org.fundacionjala.at15.pokemon;
 
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
```

Comment:
shouldn't the father class have @XmlAccessorType and the child classes just inheritance?

Reply:
Yes that's right I will change it

Reply:
I erased that annotations because it is inheriting from entity